.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Can you add .vscode to the .gitignore file please

Reply:
have added .vscode to .gitignore